app/tests/integration/test_status.py

```
@@ -0,0 +1,188 @@
+
+    # Verify that WireMock is responding
+    try:
+        async with httpx.AsyncClient(timeout=5.0) as http_client:
+            wiremock_response = await http_client.get("http://localhost:8080/v2/build")
+            assert wiremock_response.status_code == 200, \
+                f"WireMock is not responding correctly: {wiremock_response.status_code}"
+
+            wiremock_data = wiremock_response.json()
+            assert "id" in wiremock_data, "WireMock response does not have expected format"
+            print(f"✅ WireMock is working correctly: {wiremock_data}")
+    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)